ui/imports/shared/popups/keycard/KeycardPopup.qml

```diff
@@ -101,6 +105,31 @@ StatusModal {
                 return
             }
         }
+        if (root.sharedKeycardModule.currentState.flowType === Constants.keycardSharedFlow.authentication) {
```

Comment:
@stefandunca it could be grouped, but in that case it will be difficult for figuring out even for me who wrote this after some time, that's why I've splitted each flow and handle the states within each flow separately even some states are the same among flows. Tracking `root.sharedKeycardModule.currentState.flowType` you can be sure what flow we're working with.